### ISSUE TITLE
Bug generation getter/setter property FamixNG

### ISSUE THREAD

**Issue by badetitou** [pavel-krivanek/Moose-obsolete#17]
It happens when you generate a meta-model

If you want to define a property (in defineProperties) with a comment and the comment has a ' , the generation fail.
Example

```st
defineProperties
    super defineProperties.
    (javaInfixOperation property: #operator type: #String) comment: 'operator''s symbol'.
```

The class JavaInfixOperation will be generated with the setter method operator:

But with 

```st
defineProperties
    super defineProperties.
    (javaInfixOperation property: #operator type: #String) comment: 'operator s symbol'.
```

The class generated will have the setter AND the getter

**Comment by badetitou** [comment]
A quick fix could be 

```st
FmxMBTypedProperty>>#generateGetterIn: aClassOrTrait

	| methodSource propertyDefinition commentDefinition |


	propertyDefinition := '<MSEProperty: #{1} type: #{2}>' 
		format: { self name. self propertyType }.

	commentDefinition := self comment
		ifNotEmpty: [ '<MSEComment: ''{1}''>' format: { self comment replaceAllRegex: '''' with: ''''''  } ].

	methodSource := String streamContents: [ :s |
		s nextPutAll: self name; cr; cr.
		s tab; nextPutAll: propertyDefinition; cr.
		s tab; nextPutAll: '<generated>'; cr.
		self isMultivalued ifTrue: [ 
			 s tab; nextPutAll: '<multivalued>'; cr. ].
		commentDefinition  
			ifNotEmpty: [ s tab; nextPutAll: commentDefinition; cr].
		s tab; nextPutAll: '^ ';
			nextPutAll: self name.].
		
	self builder environment compile: methodSource in: aClassOrTrait classified: 'accessing'.	
```

But it is not so good... 
The bug comes from the representation of a String...

**Comment by pavel-krivanek** [comment]
fixed in d65b24d0eac4ac57677b7bcd72d6b8bcbb0b1489

**Comment by badetitou** [comment]
I check and the close the issue once everything is ok
